### PR TITLE
Added code snippet for hello world in SS_Helloworld

### PR DESCRIPTION
**Pull request by supreetsingh26**
@vanithanangunoori 
@tpchau 
@Umapaleti 
Your review?

### Todo
- [ ] sign-off before :shipit: 
- [ ] comments

### CODE REVIEW

**Comment by em-dmcgrew** [comment]
@supreetsingh26 should we schedule an all-hands meeting to discuss this?

File: SS_HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//This is a new file for testing branch and file
+
+public static void main(){
+  System.Print.Out("HelloWorld");
```
Comment (BryanNguyenEllieMae) [inline SS_HelloWorld.java]: @supreetsingh26 Let's print something new! :smile:

Comment (supreetsingh26) [inline SS_HelloWorld.java]: @BryanNguyenEllieMae What about Happy Holidays? :smiley:

**Comment by supreetsingh26** [comment]
@em-dmcgrew thats a great idea :+1:

**Pull request merged**